amostra/tests/test_jsonschema.py

```diff
@@ -0,0 +1,40 @@
+import hypothesis_jsonschema
+from hypothesis import given, settings
+from hypothesis import strategies as st
+
+from amostra.utils import load_schema
+
+sample_dict = load_schema("sample.json")
+# Pop uuid and revision cause they are created automatically
+sample_dict['properties'].pop('uuid')
+sample_dict['properties'].pop('revision')
+sample_dict['required'] = ['name']
+sample_dict['properties']['name']['minLength'] = 3
```

Comment:
Are we _required_ by hypothesis to specific a range of allowed lengths? What does it do if we leave this unset?

On the other hand, it seems reasonable to add a `minLength` to the `sample.json` schema itself --- that is, maybe we shouldn't allow the user to name something `''`. And maybe a very large `maxLength` would be reasonable too.

Whatever we choose to do, I think it would be best if the schema we generate from is as close to the actual schema as possible.

Reply:
specify a range is optional. 

I did this to get reasonable length. Cause without them, hypothesis always try to catch edge case at very beginning. What I mean is edge cases like '' and extremely long string have higher possibility to appear in a random list of strings.

Actually, I put them in test instead of in json file is notice from group meeting that how ambitious scientists is in naming and I don't want to stop them by a range. I try to limit this range locally(in test only).

I don't have strong opinion about this. Does  minLength = 1, maxLength = 50 sound good in json file?

Reply:
If it's optional, let's leave it unconstrained for now. Why should we mind if hypothesis tests edge cases? That seems good.